05-setting-up.Rmd

```diff
@@ -9,35 +9,163 @@ ottrpal::set_knitr_image_path()
 ```
 
-## Understand general principles of project organization
+## General principles of project organization
+
+Project organization should work for you and not the other way around. Maintainably organized should be the goal. Something that is effective but can be maintained long term. As you can imagine, organizational schemes are not one size fits all.
 
 ```{r, fig.align='center', out.width="100%", echo = FALSE, fig.alt= "Major point!! example image"}
-ottrpal::include_slide("https://docs.google.com/presentation/d/1MNHf8JpolaEP_vQ_kB-1xRBF9wo3haCArRu117hBoHA/edit#slide=id.g2fea8805c08_0_421")
+ottrpal::include_slide("https://docs.google.com/presentation/d/1MNHf8JpolaEP_vQ_kB-1xRBF9wo3haCArRu117hBoHA/edit#slide=id.g2fea8805c08_0_426")
 ```
 
+There's a lot of ways to keep your files organized, and there's not a "one size fits all" organizational solution [@Shapiro2021]. In this chapter, we will discuss some generalities but as far as specifics we will point you to others who have written about works for them and advise that you use them as inspiration to figure out a strategy that works for you and your team.
+
+The most important aspects of your project organization scheme is that it:  
+
+- Is [project-oriented](https://www.tidyverse.org/blog/2017/12/workflow-vs-script/) [@Bryan2017].  
+- Follows consistent patterns [@Shapiro2021].  
+- Is easy for you and others to find the files you need quickly [@Shapiro2021].  
+- Minimizes the likelihood for errors (like writing over files accidentally) [@Shapiro2021].  
+- Is something maintainable [@Shapiro2021]!
+
 ```{r, fig.align='center', out.width="100%", echo = FALSE, fig.alt= "Major point!! example image"}
-ottrpal::include_slide("https://docs.google.com/presentation/d/1MNHf8JpolaEP_vQ_kB-1xRBF9wo3haCArRu117hBoHA/edit#slide=id.g2fea8805c08_0_426")
+ottrpal::include_slide("https://docs.google.com/presentation/d/1MNHf8JpolaEP_vQ_kB-1xRBF9wo3haCArRu117hBoHA/edit#slide=id.g2fea8805c08_0_421")
+```
+
+### READMEs!
+
+READMEs are also a great way to help your collaborators get quickly acquainted with the project.
+
+```{r, fig.align='center', echo = FALSE, fig.alt= "Avi is looking at a set of project files that include a file called a ‘README.md’. Avi says 'I had no idea where to start with this analysis that Ruby sent me to review, but then I saw she included a README and that saved me so much time and effort in getting started!'"}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1LMurysUhCjZb7DVF6KS9QmJ5NBjwWVjRn40MS9f2noE/edit#slide=id.gf8379bb805_0_11")
 ```
 
+READMEs stick out in a project and are generally universal signal for new people to the project to start by READing them. GitHub automatically will preview your file called "README.md" when someone comes to the main page of your repository which further encourages people looking at your project to read the information in your README.
+
+**Information that should be included in a README:**
```

Comment:
Maybe include who is involved in the project or who to contact? (I imagine some people will not be on GitHub)

Reply:
Maybe we should talk about licenses?

Reply:
Could do it briefly

Reply:
added something below!